test/0.8.25/Protocol__Deployment.t.sol

```diff
@@ -0,0 +1,302 @@
+    address wstETH;
+}
+
+contract BaseProtocolTest is Test {
+    ILido public lidoContract;
+    ILidoLocator public lidoLocator;
+    IACL public acl;
+    SecondOpinionOracle__Mock public secondOpinionOracleMock;
+    IKernel private dao;
+
+    address private rootAccount;
+    address private userAccount;
+
+    address public kernelBase;
+    address public aclBase;
+    address public evmScriptRegistryFactory;
+    address public daoFactoryAdr;
+
+    uint256 public genesisTimestamp = 1_695_902_400;
+    address private depositContract = address(0x4242424242424242424242424242424242424242);
+    address public lidoTreasury = makeAddr("dummy-lido:treasury");
+
+    LimitsList public limitList =
+        LimitsList({
+            exitedValidatorsPerDayLimit: 9000,
+            appearedValidatorsPerDayLimit: 43200,
+            annualBalanceIncreaseBPLimit: 10_00,
+            maxValidatorExitRequestsPerReport: 600,
+            maxItemsPerExtraDataTransaction: 8,
+            maxNodeOperatorsPerExtraDataItem: 24,
+            requestTimestampMargin: 7680,
+            maxPositiveTokenRebase: 750000,
+            initialSlashingAmountPWei: 1000,
+            inactivityPenaltiesAmountPWei: 101,
+            clBalanceOraclesErrorUpperBPLimit: 50
+        });
+
+    function setUpProtocol(uint256 _startBalance, address _rootAccount, address _userAccount) public {
+        rootAccount = _rootAccount;
+        userAccount = _userAccount;
+
+        address impl = deployCode("Lido.sol:Lido");
+
+        vm.startPrank(rootAccount);
+        (dao, acl) = createAragonDao();
+        address lidoProxyAddress = addAragonApp(dao, impl);
+
+        lidoContract = ILido(lidoProxyAddress);
+
+        /// @dev deal lido contract with start balance
+        vm.deal(lidoProxyAddress, _startBalance);
+
+        acl.createPermission(userAccount, lidoProxyAddress, keccak256("STAKING_CONTROL_ROLE"), rootAccount);
+        acl.createPermission(userAccount, lidoProxyAddress, keccak256("STAKING_PAUSE_ROLE"), rootAccount);
+        acl.createPermission(userAccount, lidoProxyAddress, keccak256("RESUME_ROLE"), rootAccount);
+        acl.createPermission(userAccount, lidoProxyAddress, keccak256("PAUSE_ROLE"), rootAccount);
+
+        StakingRouter__MockForLidoAccounting stakingRouter = new StakingRouter__MockForLidoAccounting();
+
+        uint256[] memory stakingModuleIds = new uint256[](3);
+        stakingModuleIds[0] = 1;
+        stakingModuleIds[1] = 2;
+        stakingModuleIds[2] = 3;
+
+        uint96[] memory stakingModuleFees = new uint96[](3);
+        stakingModuleFees[0] = 4876942047684326532;
+        stakingModuleFees[1] = 145875332634464962;
+        stakingModuleFees[2] = 38263043302959438;
+
+        address[] memory recipients = new address[](3);
+        recipients[0] = 0x55032650b14df07b85bF18A3a3eC8E0Af2e028d5;
+        recipients[1] = 0xaE7B191A31f627b4eB1d4DaC64eaB9976995b433;
+        recipients[2] = 0xdA7dE2ECdDfccC6c3AF10108Db212ACBBf9EA83F;
```

Comment:
Not sure why this is duplicated both here and in the mock itself. What if the number is changed somewhere and we end up with a discrepancy? Maybe we should find a way to unify it.

Reply:
will optimize